### PR TITLE
add two urls to ensure correct claiming of superhero domains

### PR DESCRIPTION
**Pull request by kenodressel**

### CODE REVIEW

**Review by thepiwo — COMMENTED** [review]
looks good, can you provide a sample each so I can verify

**Comment by kenodressel** [comment]
https://superhero.com/tip/2169_v1 to claim on a tip
https://superhero.com/tip/2170_v1 to claim on a comment

File: config/regex-url-extractors.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@
 # 3. append to url before extraction
 (?<url>https:\/\/twitter.com\/[a-zA-Z0-9_]+) [data-testid="UserDescription"]
 (?<url>https:\/\/www.youtube.com\/(c|channel)\/[a-zA-Z0-9_-]+) #description /about
+(?<url>https:\/\/superhero.com\/tip\/\d+(?:_v\d+)?) .tip__body .address
```
Comment (thepiwo) [inline config/regex-url-extractors.txt:6]: the goal is to add `.address` to the url?

**Review by thepiwo — CHANGES_REQUESTED** [review]
doesn't seem to work, the provided examples still resolve the previous (incorrect) address

File: config/regex-url-extractors.txt (hunk 1 above)
Comment (kenodressel) [reply config/regex-url-extractors.txt:6]: no its supposed to be part of the CSS selector

Comment (thepiwo) [reply config/regex-url-extractors.txt:6]: this doesn't work with these selectors as currently implemented, but I can work around this

**Review by thepiwo — APPROVED** [review]

**Pull request merged**